origami.json

```
@@ -1,5 +1,6 @@
 {
-	"description": "Webfont loader for the FT's fonts.",
+	"description": "Loads CSS declarations for FT web fonts. Does not include font assets. ",
```

Trailing space, otherwise good 👍 
```suggestion
	"description": "Loads CSS declarations for FT web fonts. Does not include font assets.",
```